pages/Thread.html

```diff
@@ -0,0 +1,260 @@
+                            <div class="input-group custom-search-form">
+                                <input type="text" class="form-control" placeholder="Search...">
+                                <span class="input-group-btn">
+                                <button class="btn btn-default" type="button">
+                                    <i class="fa fa-search"></i>
+                                </button>
+                            </span>
+                            </div>
+                            <!-- /input-group -->
+                        </li>
+                        <li class="space text-center">				<!--Added the li's after the search box-->
+                            <h4>Most Discussed Topics<br><br></h4>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="1.html">1</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="2.html">2</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="3.html">3</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="4.html">4</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="6.html">5</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="6.html">6</a>
+                        </li>
+                    </ul>
+                </div>
+                <!-- /.sidebar-collapse -->
+            </div>
+            <!-- /.navbar-static-side -->
+        </nav>
+
+
+
+
+        <!--php or javascript to access all branch directories to add here-->
+        <div id="page-wrapper">
+            <div class="row">
+                <div class="col-lg-12 stick">
+                    <h1 class="page-header">Research ID<!--  can be found using php-->:</h1>
+                </div>
+                <!-- /.col-lg-12 -->
+            </div>
+            <!-- /.row -->
+            <div class="row maintain">					<!--Added .maintain -->
+                <div class="col-lg-12">
+                    <div class="panel panel-yellow">
+                        <div class="panel-heading"><h3>
+                            Topic of Research</h3>
+                        </div>
+                        <div class="panel-body">
+                            <p>Description</p>
+                        </div>
+                        <div class="panel-footer">
+                            Professors and Students involved
+                        </div>
+                    </div>
+                    <!-- /.col-lg-4 -->
+                </div>
+                <div class="col-lg-12">					<!--Added several changes in the discussion box-->
```

Comment:
Remove comment and other such unnecessary ones. Though commenting is a good practice :)